hpy/devel/include/cpython/hpy.h

```diff
@@ -49,10 +49,11 @@ typedef struct _HPyContext_s {
     HPy h_True;
     HPy h_False;
     HPy h_Exception;
-    HPy h_ValueError;
-    HPy h_TypeError;
     HPy h_IndexError;
+    HPy h_OverflowError;
     HPy h_SystemError;
+    HPy h_TypeError;
+    HPy h_ValueError;
```

Comment:
at some point we should just make a branch to add all exceptions at once.
Currently it is very fragile both here and on PyPy because you need to manually initialize these in the very same order, so whenever we do such a change the risk of breaking things is high.

Reply:
Let me do that next and get it out of the way and then it's done.